packages/forms-web-app/src/views/layouts/default.njk

```diff
@@ -4,6 +4,8 @@
 
 {%- set initiateScriptsConfig = [{ src: "/public/scripts/index.script.js" }] -%}
 
+{% set imagePath = '/public' %}
```

Comment:
As this is the image path could we include the image folder: '/public/image' or change it to publicPath to accommodate manifest.json?

Reply:
Yes, I think I'll go with your second suggestion